src/test/fuzz/span.cpp

```diff
@@ -32,7 +32,7 @@ void test_one_input(const std::vector<uint8_t>& buffer)
     }
 
     std::string another_str = fuzzed_data_provider.ConsumeBytesAsString(32);
-    const Span<const char> another_span = MakeSpan(another_str);
+    const Span<const char> another_span{str};
```

Comment:
copy paste typo? shouldn't this be:
```suggestion
    const Span<const char> another_span{another_str};
```

Reply:
Thanks, fixed!